source/1_engine/rex_engine/src/diagnostics/logging/logger_config.cpp

```diff
@@ -1,27 +1,32 @@
-
-
-#pragma once
-
-#include "rex_engine/diagnostics/logging/internal/cfg/helpers.h"
+#include "rex_engine/diagnostics/logging/logger_config.h"
```

Comment:
this file will need some adjusting after the new way of using command line args is in so I won't comment on this